client/my-sites/upgrades/cart/cart-plan-ad.jsx

```diff
@@ -18,23 +18,23 @@ import * as upgradesActions from 'lib/upgrades/actions';
 import { PLAN_PREMIUM } from 'lib/plans/constants';
 
 class CartPlanAd extends Component {
-	addToCartAndRedirect( event ) {
+	addToCartAndRedirect = ( event ) => {
 		event.preventDefault();
 		upgradesActions.addItem( cartItems.premiumPlan( PLAN_PREMIUM, { isFreeTrial: false } ) );
 		page( '/checkout/' + this.props.selectedSite.slug );
-	}
+	};
 
-	shouldDisplayAd() {
```

Comment:
Thought about it, but it uses `this`, and in the case someone does pass it, no need to worry about it. :)